plugin-packs/postcss-preset-env/src/lib/plugin-id-help.js

```diff
@@ -11,6 +11,10 @@ export function pluginIdHelp(featureNamesInOptions, root, result) {
 		const byId = mostSimilar(featureName, featureNames);
 		const byPackage = mostSimilar(featureName, packageNames);
 
+		// TODO :
+		// 1. create markdown docs with the plugin id's
```

Comment:
outside of scope of this change.
But I want to already indicate were in the code this can be mentioned to users.